### PR TITLE
Set default tag to v1.2.0

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by fewdan — APPROVED** [review]
LGTM

File: helm/chaos-mesh/values.yaml
Comment (Colstuwjx) [inline helm/chaos-mesh/values.yaml:36]: I can't find v1.2.0 image from the docker hub: https://registry.hub.docker.com/r/pingcap/chaos-mesh/tags?page=1&ordering=last_updated&name=v.1.20, should we upload it first?

Comment (cwen0) [reply helm/chaos-mesh/values.yaml:36]: At present, we update the default tag first and then build the image after this pr merged. Because I want to add this pr to v1.2.0 tag. This may be not the best practice. We will adjust the release process in the future and plan to pull a proposal that our maintainers and committers take turns to be the release manager.

**Review by Colstuwjx — APPROVED** [review]
LGTM

**Pull request merged**
